Auth0/OAuth2Grant.swift

```diff
@@ -144,7 +144,7 @@ private func decode(jwt: String) -> [String: Any]? {
     let paddingLength = requiredLength - length
     if paddingLength > 0 {
         let padding = "".padding(toLength: Int(paddingLength), withPad: "=", startingAt: 0)
-        base64 = base64 + padding
+        base64 += padding
```

Comment:
Was really just to please a SwiftLint error, I'll make an exception. I was a bit surprised that this was an `error`